### PR TITLE
Fixed errors in rawwebsocket transport

### PR DESCRIPTION
**Pull request by sharik**
1. Removed existence check for "ORIGIN" header (why it is forbidden?)
2. Process FRAME_MESSAGE_BLOB message type

### CODE REVIEW

File: sockjs/route.py [hunk 1]
```diff
@@ -152,8 +152,8 @@ def websocket(self, request):
         session = self.manager.get(sid, True, request=request)
 
         # websocket
-        if hdrs.ORIGIN in request.headers:
-            return web.HTTPNotFound()
+        # if hdrs.ORIGIN in request.headers:
```
Comment (fafhrd91) [inline sockjs/route.py]: just remove it

**Comment by fafhrd91** [comment]
did you run sockjs test suite?

**Comment by iceboy-sjtu** [comment]
I attempted to connect to sockjs using WebSocket4Net. Unfortunately it always adds a "origin" header where there was no way to remove it. ＋1 on removing this check. Also the CI failure seems like a system error which is irrelevant to the modification.

**Comment by twd2** [comment]
+1 on removing this check.

**Pull request merged**